app/models/asset_field_type.rb

```diff
@@ -36,6 +36,7 @@ def data
         'file_size': asset_file_size,
         'updated_at': asset_updated_at
       },
+      'asset_title_slug': asset_title_slug,
```

Comment:
I think this would be better written as `media_title` - `slug` seems extraneous, and it's actually the title of the `Media`, not an `asset`.
![image](https://cloud.githubusercontent.com/assets/1045168/23561065/ae319a6a-000a-11e7-9ebb-0d8cb8485f88.png)